Package.swift

```diff
@@ -14,43 +13,26 @@ let package = Package(
         .package(
             name: "swift-baggage-context",
             url: "https://github.com/slashmo/gsoc-swift-baggage-context",
-            from: "0.1.0"
+            from: "0.2.0"
         ),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.17.0"),
-        .package(url: "https://github.com/apple/swift-log.git", from: "1.2.0")
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.17.0")
     ],
     targets: [
-        // ==== --------------------------------------------------------------------------------------------------------
-        // MARK: Baggage + Logging
-
-        .target(
-            name: "BaggageLogging",
-            dependencies: [
-                .product(name: "Baggage", package: "swift-baggage-context"),
-                .product(name: "Logging", package: "swift-log"),
-            ]
-        ),
-        .testTarget(
-            name: "BaggageLoggingTests",
-            dependencies: [
-                "BaggageLogging",
-            ]
-        ),
-
         // ==== --------------------------------------------------------------------------------------------------------
         // MARK: Instrumentation
 
         .target(
             name: "Instrumentation",
             dependencies: [
                 .product(name: "Baggage", package: "swift-baggage-context"),
+                .product(name: "BaggageLogging", package: "swift-baggage-context"),
```

Comment:
does it have to?

Reply:
This was the only way I knew of to get `BaggageLogging` available in the `UseCases` executables.

Reply:
Should be able to add it in the UseCases/Package.swift :)

Reply:
How would you do that without adding a dependency on `baggage-context`? None of the following works:

- `.product(name: "BaggageLogging", package: "gsoc-swift-tracing"), // product not found`
- `.product(name: "BaggageLogging", package: "swift-baggage-context"), // unknown package`
- `.product(name: "BaggageLogging", package: "gsoc-swift-baggage-context"), // unknown package`

Reply:
Shared on chat what should work fine

Reply:
👍